source/includes/common-minio-mc.rst

```diff
@@ -1,23 +1,8 @@
 .. start-minio-mc-globals
 
-.. mc-cmd:: --debug
-   :optional:
-   
-   Enables verbose output to the console.
-
-   For example:
-
-   .. code-block:: shell
-      :class: copyable
-
-      mc --debug COMMAND
+Use any of the :ref:`global flags <minio-mc-global-options>` with this command.
```

Comment:
```suggestion
This command supports any of the :ref:`global flags <minio-mc-global-options>`.
```

or say "You can use any global flags..."

Nit picky really.

Reply:
Your suggestion is better, thanks.